lib/matplotlib/text.py

```diff
@@ -816,21 +820,20 @@ def get_stretch(self):
 
     def get_horizontalalignment(self):
         """
-        Return the horizontal alignment as string.  Will be one of
-        'left', 'center' or 'right'.
+        Return the horizontal alignment as one of {'left', 'center', 'right'}.
         """
         return self._horizontalalignment
 
     def get_unitless_position(self):
-        "Return the unitless position of the text as a tuple (*x*, *y*)"
+        """Return the (x, y) unitless position of the text."""
```

Comment:
perhaps be consistent between
```
x, y
(x, y)
(*x*, *y*)
*(x, y)*
```
?
all of them occur right now in this patch.

Reply:
Gone with the simple `(x, y)`.